README.md

```diff
@@ -1,3 +1,34 @@
-HTMl and CSS3 Module 1 project:  Positioning and Floating Elements.
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-Author: Mariano Abilleira.
+# Project Name
```

Comment:
You can change the title from "Project title" to the actual title like "HTML and CSS3 Module 1 project:  Positioning and Floating Elements" that you used before 👍🏼